src/cartesian/XAxis.js

```diff
@@ -19,12 +19,15 @@ class XAxis extends PureComponent {
     unit: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     // The unique id of x-axis
     xAxisId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    domain: PropTypes.arrayOf(PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.number,
+    domain: PropTypes.oneOfType([
```

Comment:
Cor `PropTypes` - you know it's _old_ React when you're having to define these.

Reply:
IKR, blast from the past